src/Flame/Types.purs

```diff
@@ -59,6 +61,26 @@ data Html message =
 
 derive instance elementFunctor :: Functor Html
 
+
+instance showHtml :: Show (Html message) where
+        show (Node tag nodeData childs) = "(Node " <> show tag <> " " <> show (nodeData # filter isShowable) <> " " <> show childs <> ")"
+                where isShowable  = case _ of
+                        Attribute _ _ -> true
+                        Property _ _ -> true
+                        _ -> false
+        show (Text t) = "(Text " <> show t <> ")"
+
+instance eqHtml :: Eq (Html message) where
+        eq (Node tag nodeData childs) (Node tag2 nodeData2 childs2) = tag == tag2 && eqArrayNodeData nodeData nodeData2 && childs == childs2
+                where eqArrayNodeData arr1 arr2 = all (flip elem (onlyComparable arr2)) (onlyComparable arr1)
+                      onlyComparable = filter case _ of
+                        Attribute _ _ -> true
+                        Property _ _ -> true
+                        _ -> false
```

Comment:
Maybe `onlyComparable` and `isShowable` could be abstracted?

Reply:
Should be doable. Maybe a function "isNonEventData"?

Reply:
Sure!